### PR TITLE
Create temporary dag for export all history

### PR DESCRIPTION
**Pull request by chowbao**
Create a temporary dag to execute `export_all_history` to reduce number of captive cores used within stellar-etl/Hubble

### CODE REVIEW

File: airflow_variables_prod.json [hunk 1]
```diff
@@ -154,7 +154,7 @@
   "dbt_threads": 1,
   "gcs_exported_data_bucket_name": "us-central1-hubble-2-d948d67b-bucket",
   "gcs_exported_object_prefix": "dag-exported",
-  "image_name": "stellar/stellar-etl:a480828",
+  "image_name": "chowbao/stellar-etl:ef553e5",
```
Comment (chowbao) [inline airflow_variables_prod.json:157]: TODO: Update when stellar-etl image is built

File: dags/history_archive_with_captive_core_combined_export_dag.py [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+from stellar_etl_airflow.build_gcs_to_bq_task import build_gcs_to_bq_task
+from stellar_etl_airflow.build_time_task import build_time_task
+from stellar_etl_airflow.default import get_default_dag_args, init_sentry
+
+init_sentry()
+
+dag = DAG(
+    "history_archive_with_captive_core_combined_export",
+    default_args=get_default_dag_args(),
+    start_date=datetime(2024, 1, 18, 23, 0),
```
Comment (chowbao) [inline dags/history_archive_with_captive_core_combined_export_dag.py:28]: TODO: Adjust time to when this is merged and deployed to prod

File: airflow_variables_prod.json (hunk 1 above)
Comment (sydneynotthecity) [reply airflow_variables_prod.json:157]: Is this image currently running on testnet? Just want to make sure we have a sufficient testing period before deploying to mainnet.

File: dags/history_archive_with_captive_core_combined_export_dag.py [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+from stellar_etl_airflow.default import get_default_dag_args, init_sentry
+
+init_sentry()
+
+dag = DAG(
+    "history_archive_with_captive_core_combined_export",
+    default_args=get_default_dag_args(),
+    start_date=datetime(2024, 1, 18, 22, 0),
+    catchup=True,
+    description="This DAG exports trades and operations from the history archive using CaptiveCore. This supports parsing sponsorship and AMMs.",
```
Comment (sydneynotthecity) [inline dags/history_archive_with_captive_core_combined_export_dag.py:30]:
```suggestion
    description="This DAG exports all history_* base tables using CaptiveCore. The DAG is a temporary fix, and not suited for public use.",
```

**Review by sydneynotthecity — APPROVED** [review]
LGTM

File: airflow_variables_prod.json (hunk 1 above)
Comment (chowbao) [reply airflow_variables_prod.json:157]: Yes it is currently running on testnet since last night

**Pull request merged**